src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -1,19 +1,35 @@
 import { User } from 'shared';
-import { Controller, Control, FieldErrorsImpl } from 'react-hook-form';
-import { FormControl, FormLabel, Grid, MenuItem, TextField } from '@mui/material';
+import { FormControl, FormLabel, Grid } from '@mui/material';
 import PageBlock from '../../../layouts/PageBlock';
 import ReactHookTextField from '../../../components/ReactHookTextField';
 import AttachMoneyIcon from '@mui/icons-material/AttachMoney';
 import { fullNamePipe } from '../../../utils/pipes';
-import { ProjectEditFormInput } from './ProjectEditContainer';
+import NERAutocomplete from '../../../components/NERAutocomplete';
 
 interface ProjectEditDetailsProps {
   users: User[];
-  control: Control<ProjectEditFormInput>;
-  errors: FieldErrorsImpl<ProjectEditFormInput>;
+  control: any;
```

Comment:
change these types back to what they were before (look on github) I did the same thing idk why